index.html

```diff
@@ -12,6 +12,57 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header class="navbar-container">
+        <img class="hamburger" src="/home/dom/Documents/HTML-CSS-Coursework-Week2/img/menu-hamburger.svg" width="3%" height="3%" alt="logo"><!--little logo missing-->
+        <img class="logo" src="/home/dom/Documents/HTML-CSS-Coursework-Week2/img/karma-logo.svg" width="3%" height="3%" alt="logo"><!--little logo missing-->
+        <nav>
+            <ul>
+                <li><a class="navLink" href="">Meet Karma</a></li>
+                <li><a class="navLink" href="">How it Works</a></li>
+                <li><a class="navLink" href="">Store</a></li>
+                <li><a class="navLink" href="">Blog</a></li>
+                <li><a class="navLink" href="">Help</a></li>
+                <li><a class="navLink" href="">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="main">
+        <div class="container">
+            <h2>Introducing Karma</h2>
+            <h3>Bring WiFi with you, everywhere you go.</h3>
+            <button class="cta">Learn More</button>
+        </div>
+        <section>
+            <h1>Everyone needs a little Karma.</h1>
+        </section>
+        <section>
+            <ul class="bar">
+                <li> Internet for all devices
+                    <img src="/home/dom/Documents/HTML-CSS-Coursework-Week2/img/icon-devices.svg"  width="10%" height="10%" alt="devices-icon">
+                </li>
+                <li> Boost your productivity
+                    <img src="/home/dom/Documents/HTML-CSS-Coursework-Week2/img/icon-coffee.svg" width="10%" height="10%" alt="coffe-icon">
```

Comment:
Same comment here about absolute paths. Start at `./img/________`

Reply:
Thank you Lucy. Done